models/navigation.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+type Navigation struct {
```

Comment:
(suggestion): Probably worth adding a comment on what ```Navigation``` does for its consumers to understand as this is an exported api.

Reply:
Done